bonus-jest/demo/src/TestMe.tsx

```diff
@@ -17,21 +17,25 @@ export const TestMe = (props: TestMeProps) => {
 };
 
 export class InnerMe extends React.Component<TestMeProps, TestMeState> {
+  setState(arg0: { clicked: boolean; }): any {
+    throw new Error("Method not implemented.");
+  }
   state = {
     clicked: false
   };
 
   onClick = () => {
     this.setState({ clicked: true });
   };
+  props: any;
```

Comment:
not familiar for why this is here

Reply:
There was a type error and using this rectified it.

Reply:
Are there any more changes that are required to be made or is it good to go?